### PR TITLE
traitor item Energy crossbow

### PR DESCRIPTION
**Pull request by yglop**
## About the PR
Adds energy crossbow to uplink for 50 tc.

## Why / Balance
This will add more variety and I hope that less players will take storage implant + emag and do nothing the whole shift.

## Technical details
Original commit by [Ratyyy](https://github.com/Ratyyy):
https://github.com/Workbench-Team/space-station-14/commit/e2414c73e0dc72167a68a7e2846b15de80f7b24e

## Media
Screenshot from 2024-08-13 11-25-34

https://github.com/user-attachments/assets/af6438d3-67de-4bef-813d-62ea1be6e460

**Changelog**
:cl:
- add: Syndicate weapons supply department can now provide new Energy Crossbow for just 49.99!

### CODE REVIEW

File: Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - type: Sprite
+    noRot: false
+    sprite: Goobstation/Objects/Weapons/Guns/Projectiles/cbbolt.rsi
+    layers:
+    - state: cbbolt
+  - type: GatheringProjectile
+  - type: Projectile
+    damage:
+      types:
+        Poison: 10
```
Comment (whateverusername0) [inline Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:17]: please change that to heat

**Review by whateverusername0 — CHANGES_REQUESTED** [review]

**Comment by whateverusername0** [comment]
unrelated sawmill test, merg now :trollface:

File: Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml (hunk 1 above)
Comment (yglop) [reply Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:17]: but tg wiki said 
image
:cry:

Comment (whateverusername0) [reply Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:17]: > but tg wiki said

your crossbow doesn't knock the eveny down for 1 seconds so it's ooseless
also why tf does *energy* crossbow deal *toxin* damage? this i call bad design choices

Comment (JoeHammad1844) [reply Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:17]: budget AP :trollface:

Comment (yglop) [reply Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: done

**Review by whateverusername0 — APPROVED** [review]
i approve of this PR (even though i'm not a maintainer :trollface:)

**Comment by Aidenkrz** [comment]
Can we get some media on this?

**Comment by Aidenkrz** [comment]
https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html?highlight=changelog#changelog

**Pull request merged**